spec/integration_spec.rb

```diff
@@ -365,6 +372,7 @@
     @products_in_database = Product.all
 
     Product.reindex!(MeiliSearch::Rails::IndexSettings::DEFAULT_BATCH_SIZE, true)
+    sleep 5
```

Comment:
👀

Reply:
Oh no, the rebase thought this PR wanted to add the sleep back, I probably have the same problem in all the other PRs I rebased...